### PR TITLE
Add php container

### PR DESCRIPTION
**Pull request by liujianjunhq**
Signed-off-by: Liu Jianjun <jianjun.liu@intel.com>

### CODE REVIEW

**Comment by liujianjunhq** [comment]
Thanks a lot for your reminding, and I have the toplevel README.md and .travis.yml file updated.

**Comment by gtkramer** [comment]
@liujianjunhq can you please rebase and squash your commits?

File: php/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Then, run the commands to build and run the Docker image:
+```
+$ docker build -t my-own-php-instance .
+$ docker run -it --rm --name my-running-php my-own-php-instance
+```
+
+Extra Build ARGs
+----------------
+- ``swupd_args`` Specifies [SWUPD](https://clearlinux.org/documentation/swupdate_how_to_run_the_updater.html) flags
```
Comment (phmccarty) [inline php/README.md]: Can you update this link to `https://github.com/clearlinux/swupd-client/blob/master/docs/swupd.1.rst#options` instead? See #104 for the rationale.

File: README.md [hunk 2]
```diff
@@ -19,3 +19,4 @@ Containers
 - stacks-dlrs-mkl
 - stacks-pytorch-oss
 - stacks-pytorch-mkl
+- php
```
Comment (gtkramer) [inline README.md]: This change is no longer needed.  We've removed the list of images.

File: .travis.yml [hunk 3]
```diff
@@ -15,6 +15,7 @@ env:
   - DOCKERFILE_DIR=nginx
   - DOCKERFILE_DIR=redis
   - DOCKERFILE_DIR=httpd
+  - DOCKERFILE_DIR=php
```
Comment (gtkramer) [inline .travis.yml]: Please rebase to get the current sorted list of images for the Travis CI config.

**Comment by liujianjunhq** [comment]
Will submit a new PR for this. Thanks a lot!

**Pull request closed without merging**